utils/pvacapi/config/swagger.yaml

```diff
@@ -1035,6 +1035,50 @@ paths:
           description: "Unexpected error"
           schema:
             $ref: "#/definitions/Error"
+  /api/v1/export/{processID}:
```

Comment:
In order to make this conform to REST better, this should be `/api/v1/process/{processID}/export` and should be a POST request instead of GET.